airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -385,6 +385,22 @@ private Job getJob(final DSLContext ctx, final long jobId) {
     return listJobsWithStatus(Sets.newHashSet(configType), status);
   }
 
+  @Override
+  public List<JobStatus> listJobStatusWithConnection(final UUID connectionId, final Set<ConfigType> configTypes, final Instant jobCreatedAtTimestamp)
+      throws IOException {
+    final LocalDateTime timeConvertedIntoLocalDateTime = LocalDateTime.ofInstant(jobCreatedAtTimestamp, ZoneOffset.UTC);
+
+    final String JobStatusSelect = "SELECT status FROM jobs ";
+    return jobDatabase.query(ctx -> ctx
+        .fetch(JobStatusSelect + "WHERE " +
+            "CAST(scope AS VARCHAR) = ? AND " +
```

Comment:
Why do we need to cast as a VARCHAR? `scope = '?'` should be working (same with the config type).

Reply:
I thought that was the convention since most of the other queries casted the scope, but just ran the unit tests without it and had no problems, so i'll remove it.

Reply:
I misspoke earlier, looks like the convention is not to cast the scope. But `ConfigType` is normally casted.